packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -3,6 +3,23 @@ title: Okta API Products Release Notes 2021
 ---
 ## August
 
+### Weekly Release 2021.08.2
+
+| Change                                                                     | Expected in Preview Orgs |
+|----------------------------------------------------------------------------|--------------------------|
+| [Custom Administrator Roles is Self-Service EA](#custom-administrator-roles-is-self-service-ea) | August 18, 2021 |
+| [Bugs fixed in 2021.08.2](#bugs-fixed-in-2021-08-2)                          | August 18, 2021          |
+
+#### Custom Administrator Roles is Self-Service EA
+
+The [Custom Administrator Roles](/docs/reference/api/roles/index.md) feature is now enabled as Self-Service Early Access. The Okta Custom Administrator Roles API provides operations so that you can create customized roles and assign them to a user or a group. <!--OKTA-419528-->
```

Comment:
avoid repeating exactly the same info that is in the heading.
```suggestion
The Okta [Custom Administrator Roles](/docs/reference/api/roles/index.md) API provides operations so that you can create customized roles and assign them to a user or a group. <!--OKTA-419528-->
```